pkg/distributor/distributor_test.go

```diff
@@ -576,14 +576,14 @@ func (i *mockIngester) QueryStream(ctx context.Context, req *client.QueryRequest
 
 		c := encoding.New()
 		for _, sample := range ts.Samples {
-			cs, err := c.Add(model.SamplePair{
+			// TODO(codesome): consider additional chunks.
```

Comment:
This comment doesn't mean much to me.

Reply:
That was a TODO for me which I forgot to fix. I have updated it now to handle any additional chunks.